src/Okta.Idx.Sdk/IAuthenticatorEnrollment.cs

```diff
@@ -57,5 +58,10 @@ public interface IAuthenticatorEnrollment : IResource
         /// Gets the credential Id. This is only applicable for Web Authn.
         /// </summary>
         string CredentialId { get; }
+
+        /// <summary>
+        /// Gets the contextual data used by Okta Verify enrollment.
+        /// </summary>
+        IOktaVerifyContextualData OktaVerifyContextualData { get; }
```

Comment:
We should integrate this with the current approach. There's a generic AuthenticatorContextualData property already, and both WebAuthn and Google Authn have been integrated to use the same. We should also do that with Okta Verify.

Reply:
I had done that originally, but when resolving conflicts things had changed causing breakage.  I'll review again, to integrate as you describe.

Reply:
@andriizhegurov-okta and I can help you with this. We had to work together to integrate our code.